azext_edge/edge/vendor/clients/assetmgmt/_client.py

```diff
@@ -0,0 +1,134 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
```

Comment:
Not sure if a an entire separate client is necessary if there are no breaking changes between versions. api_version kwargs can be passed to override API versions on the client